src/utils/KML.js

```diff
@@ -185,15 +183,37 @@ const sanitizeFeature = (feature) => {
     fill = undefined;
     stroke = undefined;
 
-    styles = [
-      new Style({
-        fill,
-        stroke,
-        image,
-        text,
-        zIndex: style.getZIndex(),
-      }),
-    ];
+    styles = (f, resolution) => {
+      /* Options to be used for picture scaling with map, should have at least
+       * a resolution attribute (this is the map resolution at the zoom level when
+       * the picture is created), can take an optional constant for further scale
+       * adjustment.
+       * e.g. { resolution: 0.123, constant: 1 / 6 }
+       */
+      if (feature.get('pictureOptions') && mapResolution) {
+        let pictureOptions = feature.get('pictureOptions');
+        if (typeof pictureOptions === 'string') {
+          pictureOptions = JSON.parse(feature.get('pictureOptions'));
+        }
+        feature.set('pictureOptions', pictureOptions);
+        if (pictureOptions.resolution) {
+          image.setScale(
+            (pictureOptions.resolution / resolution) *
+              pictureOptions.constant || 1,
```

Comment:
call the property 'defaultScale' or 'ratio' . I don't know what is best , 'constant ' is too generic.

Reply:
Set to defaultScale